From af4deff667548292e51222dbdcdfb06f83900e6a Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Wed, 30 Nov 2022 10:49:54 +0100
Subject: [PATCH 0302/1695] dts: stm32: enable risaf2/5 nodes for
 stm32mp257f-ev1

Enables risaf2 and risaf5 nodes for this board.
These RISAF instances restrict accesses to OSPI and PCIE respectively.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 70f77e2bbffd3918e42cc42c93257eb4e9e7ed4f dts: stm32: enable risaf2/5 nodes for stm32mp257f-ev1]
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: I5d9c5ec4206d407727e84c2e6746042f73bf1fb9
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/278354
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/dts/stm32mp257f-ev1.dts | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp257f-ev1.dts b/core/arch/arm/dts/stm32mp257f-ev1.dts
index 1198bac3b..897b593da 100644
--- a/core/arch/arm/dts/stm32mp257f-ev1.dts
+++ b/core/arch/arm/dts/stm32mp257f-ev1.dts
@@ -37,6 +37,14 @@
 	};
 };
 
+&risaf2 {
+	status = "okay";
+};
+
+&risaf5 {
+	status = "okay";
+};
+
 &usart2 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&usart2_pins_a>;
-- 
2.47.2

